Sources/Features/DappInteractionFeature/Coordinator/DappInteractionFlow.swift

```diff
@@ -43,11 +43,13 @@ struct DappInteractionFlow: Sendable, FeatureReducer {
 		let interactionItems: NonEmpty<OrderedSet<AnyInteractionItem>>
 		var responseItems: OrderedDictionary<AnyInteractionItem, AnyInteractionResponseItem> = [:]
 
+		var interactionItem: AnyInteractionItem? = nil
```

Comment:
You might want to look at [this PR](https://github.com/davdroman/tca-relay-reducer/pull/1) which [computes the current request](https://github.com/davdroman/tca-relay-reducer/pull/1#discussion_r1391866388) (interaction item) based on the current path instead of holding onto it as state which is more prone to bugs by virtue of having to mutate it at different points during the execution flow.

Reply:
Thanks, I'll definitely have a look. Trying to get rid of the fork is something I want to do this week or next so this comes in handy!